franklin/builder/models.py

```diff
@@ -47,12 +47,14 @@ def save(self, *args, **kwargs):
         base_path = os.environ['BASE_PROJECT_PATH']
         self.path = "{0}/{1}".format(base_path, self.repo_name)
         if not self.url:
+            repo_name = self.repo_name.split("/")[1]
             self.url = "{0}.{1}".format(
-                self.repo_name.replace('/', '-'),
+                repo_name,
                 os.environ['BASE_URL']
             )
         super(Site, self).save(*args, **kwargs)
-        self.build()
+        if os.environ['ENV'] is 'production':
```

Comment:
We need a way to run tests without triggering a build...maybe this is the right way of doing it? Interested in suggestions.